frog/imports/api/wikiDocHelpers.js

```diff
@@ -117,6 +117,14 @@ export const createNewEmptyWikiDoc = (wikiDoc, wikiId, liId, owner) => {
   const emptyDocValues = {
     wikiId,
     owners: Array.of(owner),
+    users: Array.of(owner),
+    editors: Array.of(owner),
+    settings: {
+      password: '',
```

Comment:
Implemented on the suggestion of @houshuang , we need to show the existing password to the user in case the owner has shared it with some subset of users and forgotten it. In this case, the owner would not want to change the password since it would cause that subset of users with the existing password to be communicated of this change.

Reply:
Okay, let's make this clear in a code comment